drivers/flash/Kconfig

```diff
@@ -36,6 +36,17 @@ config FLASH_PAGE_LAYOUT
 	help
 	  Enables API for retrieving the layout of flash memory pages.
 
+config FLASH_EMULATOR
+	bool "Emulated flash driver"
+	depends on FLASH
+	default n
```

Comment:
This seems to be common in Zephyr, and it doesn't hurt, so this is more of a style note:

`bool` symbols implicitly default to `n`, so a lone `default n` is redundant.